src/logger/loggerTaskEx.c

```diff
@@ -51,7 +51,7 @@
 #define BACKGROUND_SAMPLE_RATE	SAMPLE_50Hz
 
 int g_loggingShouldRun;
-int g_configChanged;
+int g_config_changed;
 int g_telemetryBackgroundStreaming;
```

Comment:
Did you intend to change int to bool as you changed assignments from 0/1 to false/true.  I can't actually tell what size gcc is using for a bool, but it is a bit more consistent.  (Looking stdbool.h it defines false/true to 0/1) so its functionally pretty much the same.

Reply:
General refactoring of old code.  The variable was always intended as bool. :+1:

Reply:
I really meant - it looks like we are using "bool" elsewhere rather than "int".  I'm hoping that "bool" is of type "unsigned char" rather than "int".  I'd encourage changing the type for this variable too.

Reply:
Ah, got it.  Yes, I did want to change the type too, good catch. Fixed now.